doc/sphinx/proofs/automatic-tactics/auto.rst

```diff
@@ -220,41 +206,40 @@ Programmable proof search
    This tactic solves goals that belong to many common classes; in particular, many cases of
    unsatisfiable hypotheses, and simple equality goals are usually solved by this tactic.
 
-.. tacv:: now @tactic
-   :name: now
+.. tacn:: now @ltac_expr
 
    Run :n:`@tactic` followed by :tacn:`easy`. This is a notation for :n:`@tactic; easy`.
 
-Controlling automation
---------------------------
-
 .. _thehintsdatabasesforautoandeauto:
 
 The hints databases for auto and eauto
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+--------------------------------------
 
 The hints for :tacn:`auto` and :tacn:`eauto` are stored in databases. Each database
-maps head symbols to a list of hints.
-
-.. cmd:: Print Hint @ident
+maps head symbols to a list of hints.  Use the :cmd:`Print Hint` command to view
+the database.
 
-   Use this command
-   to display the hints associated to the head symbol :n:`@ident`
-   (see :ref:`Print Hint <printhint>`). Each hint has a cost that is a nonnegative
-   integer, and an optional pattern. The hints with lower cost are tried first. A
-   hint is tried by :tacn:`auto` when the conclusion of the current goal matches its
-   pattern or when it has no pattern.
+Each hint has a cost that is a nonnegative
+integer and an optional pattern. Hints with lower costs are tried first.
+:tacn:`auto` tries a hint when the conclusion of the current goal matches its
+pattern or when the hint has no pattern.
 
 Creating Hint databases
-```````````````````````
+-----------------------
+
+Hint databases can be created with the :cmd:`Create HintDb` command or implicitly
+by adding a hint to an unknown database.  We recommend you always use :cmd:`Create HintDb`
+and then imediately use :cmd:`Hint Constants` and :cmd:`Hint Variables` to make
+those settings explicit.
 
-One can optionally declare a hint database using the command
-:cmd:`Create HintDb`. If a hint is added to an unknown database, it will be
-automatically created.
+Note that the default transparency
+settings differ between these two methods of creation.  Databases created with
+:cmd:`Create HintDb` have the default setting `Transparent` for both `Variables`
+and `Constants`, while implicitly created databases have the `Opaque` setting.
 
-.. cmd:: Create HintDb @ident {? discriminated}
+.. cmd:: Create HintDb @ident {? discriminated }
 
-   This command creates a new database named :n:`@ident`. The database is
+   Creates a new hint database named :n:`@ident`. The database is
    implemented by a Discrimination Tree (DT) that serves as an index of
    all the lemmas. The DT can use transparency information to decide if a
    constant should be indexed or not
```

Comment:
The reference one line below doesn't seem to make sense anymore since it refers to the section we are currently in. Could you remove it?